src/sentry/integrations/vsts/integration.py

```diff
@@ -196,6 +208,20 @@ class VstsIntegrationProvider(IntegrationProvider):
         'height': 800,
     }
 
+    def post_install(self, integration, organization):
+        unmigratable_repos = self \
+            .get_installation(integration, organization.id) \
+            .get_unmigratable_repositories()
+
+        Repository.objects.filter(
+            organization_id=organization.id,
+            provider='visualstudio',
+        ).exclude(
+            id__in=[r.id for r in unmigratable_repos],
```

Comment:
oops didn't think of this before, but since unmigratable_repos is a queryset, i think you could change this to be `id__in=unmigratable_repos.values_list('id', flat=True)` which i _think_ should make that a subquery rather than executing the unmigrateable repos query separately. then i think all of that could just be a single query rather than two.

Reply:
Apparently it'll just know what to do with `id__in=unmigratable_repos`. 👌

Reply:
Turns out Mysql can't handle the query Django creates from that, so it needs to go back to updating them individually 😒